rest_social_auth/views.py

```diff
@@ -175,7 +176,8 @@ def get_provider_name(self, input_data):
 
     def respond_error(self, error):
         if isinstance(error, Exception):
-            self.log_exception(error)
+            if not isinstance(error, AuthException) or LOG_AUTH_EXCEPTIONS:
```

Comment:
Am I right, that this is needed not to spam sentry for example?
Also, why not include `AuthException`